SeaPublicWebsite/wwwroot/css/_simple-energy-advice-custom.scss

```diff
@@ -311,6 +311,35 @@
     font-weight: 700;
 }
 
+.sea-download-button {
```

Comment:
We presumably don't need all this CSS anymore - we should make sure we have only the css we need

Reply:
We should also be able to remove the download-icon.png file from wwwroot/assets/images

Reply:
Done!